R/plotStorms.R

```diff
@@ -313,22 +316,24 @@ plotStorms <- function(sts,
   }
 
   # Plotting track(s) and labels
-  lapply(stsAux, plotTrack)
+  lapply(stsAux, plotTrack, sts@scale, sts@scalePalette)
   if (labels) {
     lapply(stsAux, plotLabels, by, pos)
   }
 
   # Adding legends
   if (legends != "none") {
-    leg <- c("TD", "TS", "Cat. 1", "Cat. 2", "Cat. 3", "Cat. 4", "Cat. 5")
-    col <- c("#00CCFF", "#00CCCC", "#FFFFB2", "#FECC5C", "#FD8D3C", "#F03B20", "#BD0026")
-
-    lty <- rep(0, 7)
-    pch <- rep(19, 7)
-    lwd <- rep(1, 7)
+    
+    
+    leg <- names(sts@scalePalette)
```

Comment:
What if `scalePalette` has no names defined ? We should defined names by default when they are not provided (`Cat. i` should be fine)

Reply:
Also, still in this idea to provide a "quick and dirty" plot when a user defines a new scale, and as already mentioned before, it would be good that `scalePalette` gets a default value based on a continous color scale and the number of intervals of `scale`

Reply:
> What if `scalePalette` has no names defined ? We should defined names by default when they are not provided (`Cat. i` should be fine)

Done in last commit

Reply:
> Also, still in this idea to provide a "quick and dirty" plot when a user defines a new scale, and as already mentioned before, it would be good that `scalePalette` gets a default value based on a continous color scale and the number of intervals of `scale`

Almost done, I still don't know how to generate a dynamic color palette based on the length of `scalePalette` without using another library. Maybe we should consider adding another dependency, as I also need to generate a dynamic color palette for the new `plotTemporal()` function. What do you think ?

Reply:
Done in last commit, try this code:

```
sds <- defStormsDataset(verbose = 0)

# Default scale palette
sts <- defStormsList(sds = sds,
                     loi = "Vanuatu",
                     names = "PAM",
                     scale = seq(0,80,5),
                     verbose = 0)

plotStorms(sts)

# Default scale is sshs
sts <- defStormsList(sds = sds,
                     loi = "Vanuatu",
                     names = "PAM",
                     verbose = 0)
```